llama-index-integrations/llms/llama-index-llms-gemini/llama_index/llms/gemini/base.py

```diff
@@ -125,6 +126,11 @@ def __init__(
             config_params["client_options"] = {"api_endpoint": api_base}
         if transport:
             config_params["transport"] = transport
+        if default_headers:
+            default_metadata: Sequence[Dict[str, str]] = []
+            for key, value in default_headers.items():
+                default_metadata.append((key, value))
+            config_params["default_metadata"] = default_metadata
```

Comment:
I'm not a `genai` expert and it took me a while to figure out why something called `default_metadata` would end up becoming HTTP headers 😅 so I thought a comment would help here.

```suggestion
            # `default_metadata` contains (key, value) pairs that will be sent with every request.
            # When using `transport="rest"`, these will be sent as HTTP headers.
            config_params["default_metadata"] = default_metadata
```